### DISCUSSION TITLE
PHP package generates warning when used with multiple repositories

### DISCUSSION THREAD

**Discussion by internalsystemerror** [renovatebot/renovate#18642] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

34.4.0

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### If you're self-hosting Renovate, tell us what version of the platform you run.

gitlab.com

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

My previous changes to include PHP (https://github.com/renovatebot/renovate/pull/18474) have introduced a new bug that is reusing the composer registry URLs and so generated a warning when there is more than one.

### Relevant debug logs

<details><summary>Logs</summary>

```
WARN: Excess registryUrls found for datasource lookup - using first configured only (repository=creativecow/carpetsnow/application-api)
       "datasource": "github-tags",
       "depName": "php",
       "registryUrls": [
         "https://gitlab.com/api/v4/group/<redacted>/-/packages/composer",
         "https://packagist.org/"
       ]
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by viceice** [comment]
It's not a bug. it's a configuration error. Please provide a config sample which reproduces the error.

**Reply by internalsystemerror** [reply]
I very much doubt that this is a configuration error as this is a result of the PR mentioned in the issue. It's including the registry URLs from `composer.json` not from any renovate configuration? If you don't have a private repository added in `composer.json` then only the packagist URL is used for `registryURLs` (and so not giving the warning) when in reality this should be empty for the `php` package.

**Reply by viceice** [reply]
ok, i see. fixing it now

**Reply by internalsystemerror** [reply]
Tx, and if you don't get time to, I was happy to try and fix this myself.

**Reply by viceice** [reply]
- #18643

**Comment by viceice** [comment, marked as answer]
- #18643
